lib/normalize.js

```diff
@@ -32,6 +33,16 @@ var _splitLocator = function (nemo, locatorString) {
 module.exports = function normalize(nemo, _locator) {
   var locator = _locator;
   var normalizedLocator;
+
+  if (locator.using && locator.value && nemo.wd.By[locator.using]) {
+    normalizedLocator = nemo.wd.By[locator.using](locator.value);
+    if(normalizedLocator.constructor === locator.constructor){
+      return locator; //input object was a normalized locator.  Just return it back
+    } else {
+      return normalizedLocator;
+    }
+  }
+
   if (locator.constructor !== String) {
```

Comment:
These errors are thrown if the function input object is not valid.  They will throw false positives if we remove the "already normalized" check (lines 37 - 44).  I think of the "already normalized" check as input validation so I think it makes sense to have it first, so that we don't have to repeat the conditional on line 37 twice.